### PR TITLE
added new medicine reagent, haloperidol.

### PR DESCRIPTION
**Pull request by markressler**
:cl: Ressler
rscadd: Adds the reagent Haloperidol.
rscadd: It is an anti-drug reagent, good for stopping assistants hyped up on meth, bath salts, and equally illegal drugs.
rscadd: It can be made with Chlorine, Fluorine, Aluminum, Potassium Iodide, and Oil.
/:cl:

### CODE REVIEW

**Comment by Iamgoofball** [comment]
emot-toot
congrats on the first PR

:+1: pretty useful chem for dealing with shitty assistants as security

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 1]
```diff
@@ -962,3 +962,26 @@ datum/reagent/medicine/syndicate_nanites/on_mob_life(mob/living/M)
 	M.adjustCloneLoss(-3*REM)
 	..()
 	return
+
+/datum/reagent/medicine/haloperidol
+	name = "Haloperidol"
+	id = "haloperidol"
+	description = "Increases depletion rates for most stimulating/hallucinogenic drugs by 5. Reduces druggy effects and jitteriness. Severe stamina regeneration penalty, causes drowsiness. 20% chance of +1 BRAIN."
```
Comment (duncathan) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: I don't like having specific values in the description. Is that standard across reagents?

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 2]
```diff
@@ -962,3 +962,26 @@ datum/reagent/medicine/syndicate_nanites/on_mob_life(mob/living/M)
...
+	reagent_state = LIQUID
+	color = "#27870a"
+	metabolization_rate = 0.4 * REAGENTS_METABOLISM
+
+/datum/reagent/medicine/haloperidol/on_mob_life(mob/living/M)
+	for(var/datum/reagent/drug/R in M.reagents.reagent_list)
+		if(R != src)
```
Comment (duncathan) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: This check isn't needed, as src will always fail the type check.

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm (hunk 1 above)
Comment (phil235) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: Yeah, we don't want gamey description. Also the text says "Severe stamina regeneration penalty" but the code actually reduce stamina damage.

Comment (tkdrg) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: Are these even displayed in the game anywhere? But yeah, I guess no harm in staying consistent.

Comment (Dorsisdwarf) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: Yes

**Comment by tkdrg** [comment]
Looks fine to me.

**Pull request merged**